docs/getting-started/publishing-data.md

```diff
@@ -40,20 +46,20 @@ $ data push mydata.csv
 The output will be:
 
 ```cli-output
-\> 🙌  your data is published!
+🙌  your data is published!
 
-\> 🔗  https://datahub.io/username/dataset (copied to clipboard)
+🔗  https://datahub.io/username/dataset/v/1 (copied to clipboard)
 ```
 
-**Note:** *by default, findability flag for your dataset is set to **unlisted**, meaning nobody else is able to see it, except you. Use **published** flag to make it publicly available*
+**Note:** *by default, findability flag for your dataset is set to **unlisted**, meaning it will not appear in search results. Use **published** flag to make it publicly available*
```

Comment:
I don't think the user would know what findability is at all. just say `by default, it is unlisted, meaning...`
User does not need to know about findability at all